src/fastoad/notebooks/02_explanation/03_scipy.ipynb

```diff
@@ -0,0 +1,299 @@
+{
```

Comment:
<blockquote>In order to solve the MDA, we will used the <code>fsolve</code> function from the <code>scipy.optimize</code> package (documentation is available <a href="https://docs.scipy.org/doc/scipy/index.html" target="_blank">here</a>)</blockquote><p>After the previous paragraph that links to Scipy main page, one click away from the documentation, I would have expected the "here" to link to doc of <code>scipy.optimize</code> (https://docs.scipy.org/doc/scipy/reference/optimize.html). </p><p>Also, it looks strange to me that only the second fsolve links to its documentation.  Besides, Jupyter Lab does not show that fsolve is a link, because of the fixed-width formatting, so it will be easily overlooked. Maybe "fsolve" should be put in normal text , or also associated to a parenthesis "(documentation here)".</p><p> </p><blockquote>... and feed ir to the <code>fsolve</code> function.</blockquote><p>feed i<strong>n</strong></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fast-aircraft-design/FAST-OAD/pull/426/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Feed it*, my missed stake.
Changed in dfe233f7